docs/security/assets/001-kyma-authorization.svg

```diff
@@ -1,2 +1,3 @@
k; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #ffffff; white-space: nowrap; ">OIDC<br />configuration<br /></div></div></div></foreignObject><text x="580" y="359" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">OIDC...</text></switch></g><path d="M 762.5 200 L 717.52 95.85" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 715.44 91.03 L 721.43 96.07 L 717.52 95.85 L 715.01 98.84 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="all"/><path d="M 847.5 200 L 902.05 95.64" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 904.48 90.99 L 904.34 98.82 L 902.05 95.64 L 898.14 95.57 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="all"/><ellipse cx="805" cy="365" rx="15" ry="15" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 28px; height: 1px; padding-top: 365px; margin-left: 791px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff" style="font-size: 17px">4</font></div></div></div></foreignObject><text x="805" y="369" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">4</text></switch></g><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 464px; margin-left: 702px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; background-color: #FFFFFF; white-space: nowrap; ">forward request</div></div></div></foreignObject><text x="702" y="468" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">forward request</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Why Environment + Namespace?
![image](https://user-images.githubusercontent.com/40655785/75434668-37686400-5952-11ea-85bf-ab1df37f2dad.png)

Reply:
Seems this is deprecated, deleting :)